### PR TITLE
Refactor onRefreshData to be controlled from one place

### PR DESCRIPTION
**Pull request by rvsia**
onRefreshData is declared only on one place. This makes the function consistent (there should be no bugs that some attributes are missing when the function is called from different places) and it makes all future changes much more manageable.

### CODE REVIEW

File: packages/inventory/src/components/table/InventoryList.js [hunk 1]
```diff
@@ -41,44 +40,18 @@ const ContextInventoryList = ({ showHealth, onRefreshData, ...props }) => {
...
-                    ...options
-                },
-                showTags,
-                getEntities
-            )
-        );
-    };
-
-    if (ref) { ref.current = { onRefreshData }; }
+    if (ref) { ref.current = { onRefreshData: (params, disableRefresh = true) => onRefreshData(params, disableRefresh) }; }
```
Comment (karelhala) [inline packages/inventory/src/components/table/InventoryList.js:54]: I would span this over a couple lines
```suggestion
    if (ref) {
      ref.current = {
        onRefreshData: (params, disableRefresh = true) => onRefreshData(params, disableRefresh)
      };
    }
```

File: packages/inventory/src/components/table/InventoryTable.js [hunk 2]
```diff
@@ -56,6 +58,50 @@ const InventoryTable = forwardRef(({
     const sortBy = useSelector(({ entities: { sortBy: invSortBy } }) => (
         hasItems ? propsSortBy : invSortBy
     ), shallowEqual);
+    const dispatch = useDispatch();
+    const store = useStore();
+
+    /**
+     * If conumer wants to change data they can call this function via component ref.
```
Comment (karelhala) [inline packages/inventory/src/components/table/InventoryTable.js:65]:
```suggestion
     * If consumer wants to change data they can call this function via component ref.
```

**Review by karelhala — APPROVED** [review]
Looking good!

File: packages/inventory/src/components/table/InventoryList.js (hunk 1 above)
Comment (rvsia) [reply packages/inventory/src/components/table/InventoryList.js:54]: I want to move this logic to the upper component in the follow-up pr.

(And also, we should set eslint if really want to change formatting. 😄  )

**Pull request merged**